README.md

```diff
@@ -43,7 +45,9 @@ OR
 
 Include **'therubyracer'** in your Gemfile.
 
-    gem "therubyracer", :require => 'v8'
+```bash
```

Comment:
@SuriyaaKudoIsc That applies to the above commands (`bundle` and `rails server`), but I don't believe that applies to `gem "therubyracer", :require => 'v8'`. That line belongs in the `Gemfile`, which is Ruby.

The CLI command to install gems can take commands like `gem install therubyracer`, but that's not what's being displayed here in the docs.